app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailPresenter.java

```diff
@@ -54,7 +55,10 @@ public void resendVerificationMail() {
                 .doOnSubscribe(disposable -> getView().showProgress(true))
                 .doFinally(() -> getView().showProgress(false))
                 .subscribe(resendMailResponse -> getView().showSnackbar("Verification Mail Resent"),
-                    throwable -> getView().showError(ErrorUtils.getMessage(throwable).toString())));
+                    throwable -> {
+                        getView().showError(ErrorUtils.getErrorDetails(throwable).toString());
+                        Timber.e(throwable, "An exception occurred : %s", throwable.getMessage());
```

Comment:
Still not handling anything

Reply:
Can you please explain a bit about what change is required?